dotnet/src/Extensions/TemplateEngine.Basic/Extensions/BasicPromptTemplateFactoryExtensions.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel.Orchestration;
+
+#pragma warning disable IDE0130
+// ReSharper disable once CheckNamespace - Using the namespace of IKernel
+namespace Microsoft.SemanticKernel.TemplateEngine.Basic;
+#pragma warning restore IDE0130
+
+/// <summary>
+/// Class for extensions methods associated with the the basic prompt template factory.
+/// </summary>
+public static class BasicPromptTemplateFactoryExtensions
```

Comment:
BasicPromptTemplateFactory is public.  Why have this as an extension method on BasicPromptTemplateFactory rather than just making it an instance method on the type?

Or should this instead be an extension method for any IPromptTemplateFactory rather than being specific to BasicPromptTemplateFactory?

Reply:
Taking Sergey's feedback below and I am removing this